backend/src/main/java/com/bakdata/conquery/models/forms/managed/RelativeFormQueryPlan.java

```diff
@@ -99,24 +115,24 @@ private SubResult executeSubQuery(QueryExecutionContext ctx, FeatureGroup featur
 		return new SubResult((MultilineContainedEntityResult) sub.execute(ctx, entity));
 	}
 
-	private void setOutcomeValues(Object[] result, Object[] value, int featureLength) {
+	private void setFeatureValues(Object[] result, Object[] value) {
 		// copy everything up to including index
-		for (int i = 0; i < 2; i++) {
+		for (int i = 0; i < 3; i++) {
 			result[i] = value[i];
 		}
 		// copy daterange
-		result[3] = value[2];
-		System.arraycopy(value, 3, result, 1 + featureLength, value.length - 3);
+		result[3] = value[3];
+		System.arraycopy(value, 4, result, 5, value.length - 4);
```

Comment:
Agree with @awildturtok here, these numbers are quite magic, an enum would make this a lot easier to understand